addons/website/models/website.py

```diff
@@ -105,7 +105,7 @@ def _default_logo(self):
     cdn_activated = fields.Boolean('Content Delivery Network (CDN)')
     cdn_url = fields.Char('CDN Base URL', default='')
     cdn_filters = fields.Text('CDN Filters', default=lambda s: '\n'.join(DEFAULT_CDN_FILTERS), help="URL matching those filters will be rewritten using the CDN Base URL")
-    partner_id = fields.Many2one(related='user_id.partner_id', relation='res.partner', string='Public Partner', readonly=False)
+    partner_id = fields.Many2one(related='user_id.partner_id', string='Public Partner', readonly=False)
```

Comment:
Or as positional argument
```suggestion
    partner_id = fields.Many2one('res.partner', related='user_id.partner_id', string='Public Partner', readonly=False)
```